editor/app/src/main/AndroidManifest.xml

```diff
@@ -26,8 +27,25 @@
                 android:name="android.support.FILE_PROVIDER_PATHS"
                 android:resource="@xml/file_paths"/>
         </provider>
+
+        <meta-data
+            android:name="com.google.android.geo.API_KEY"
+            android:value="AIzaSyCK9YRxRu0DT4vLzkfiOew1lSVBQs4LHas"/>
+        <meta-data android:name="com.google.android.gms.version"
+            android:value="@integer/google_play_services_version" />
     </application>
     <uses-permission android:name="android.permission.CAMERA" />
+
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
Internet seems reasonable, but I don't think we need external storage, locations and network state

Reply:
According to the tutorial which I followed all of these permissions are needed except for the network state one!